devops/templates/post-upload-validation.yml

```diff
@@ -0,0 +1,63 @@
+# Stage template to validate uploaded packag from PyPI
+
+# Downloads fairlearn from the specified PyPI (Test or Prod) at the version
```

Comment:
Do we want to write Fairlearn with uppercase F? I'm not sure...

Reply:
Good point